### DISCUSSION TITLE
many slides are not available

### DISCUSSION THREAD

**Discussion by g-i-o-r-g-i-o** [Lightning-AI/dl-fundamentals#9] [Lecture videos and quizzes] [answered]
Hello, there are some authorization problems on some slides: authorization required.

thanks in advance

![image](https://user-images.githubusercontent.com/9282332/207386611-5b07d6f0-dd60-461b-834b-07b9d25983c8.png)

**Comment by rasbt** [comment, marked as answer]
Sorry about that! I just see that the links are currently pointing to my personal files on Google Drive. Our web dev will update this to host the files on our server directly so that there won't be a permission issue anymore. Will let you know once that's fixed!

**Reply by rasbt** [reply]
@GianniGi The share links to the Google Drive slides should be updated now. I hope it works (it's a bit tricky for me to check, because as the creator I myself do have the permission to access these on Google Drive). If not, I will ping our webdev to refresh everything

**Reply by angellmethod** [reply]
Everything looks good to me except for both slide sets in 1.7. I think you should be able to check the behavior in a private window?

**Reply by rasbt** [reply]
Thanks, the problem is that the slide links are not visible when not logged in (e.g., in a private window). What I can try is logging out of Google. Will double check again and ping our webdev. In the meantime, let me drop the slides here: 

[1.7-model-eval-part2__slides.pdf](https://github.com/Lightning-AI/dl-fundamentals/files/10296297/1.7-model-eval-part2__slides.pdf)
[1.7-model-eval-part1__slides.pdf](https://github.com/Lightning-AI/dl-fundamentals/files/10296298/1.7-model-eval-part1__slides.pdf)

**Reply by angellmethod** [reply]
I see - I was regular browsing mode (eg. logged in) and used 'open in incognito window' to test.

**Reply by rasbt** [reply]
Ah yes, that would work to! Good point.

**Reply by amf60** [reply]
Hello Sebastian, first THANK YOU!! Suggesting correction - slide 19 - 1.7 model part 2: Confusion Matrix:  [2, 2] = 2 not 3.
![Screen Shot 2023-01-06 at 08 49 45](https://user-images.githubusercontent.com/39899585/211006897-e489ce34-e217-4c38-b21c-5a553f362d6e.png)

**Reply by rasbt** [reply]
Wow, good catch, and thanks a lot @amf60. I added it to the [errata](https://github.com/Lightning-AI/dl-fundamentals/tree/main/errata) for now but will update the slides and video (probably in February once the recording of units 8-10 is done).
